roles/lib/files/FWO.Api.Client/Data/NetworkLocation.cs

```diff
@@ -1,14 +1,35 @@
-using System.Text.Json.Serialization; 
+using System.Text.Json.Serialization;
 using Newtonsoft.Json;
 
 namespace FWO.Api.Data
 {
     public class NetworkLocation
```

Comment:
Should be derived from interface IComparable

Reply:
done ;-)